src/utils/regular-expression.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Utility function that will escape all regular expression special characters in a string.
+ *
+ * @param text The string potentially containing special characters.
+ * @returns The string with all special characters escaped.
+ */
+export const escapeRegExpSpecialCharacters = (text: string) => {
```

Comment:
Made this a util since I could see this coming up again. If something like this already exists, or we do something similar in other places, I can update accordingly

Reply:
I don't know of anything but this looks like it will be handy in the future!